__tests__/users.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require('../app');
+const request = require('supertest');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const data = require('../db/data/test-data');
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/users", () => {
+    describe("GET", () => {
+        test("Status 200 - Returns array of user objects", () => {
+            return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then(({body}) => {
+                expect(typeof body).toBe("object");
```

Comment:
To be sure, to check for an object you are best to use the matcher toBeInstanceOf(Object) 

In your example an array would also pass this test.